### ISSUE TITLE
Is there any chance of any kind of documentation?

### ISSUE THREAD

**Issue by eahogue** [cu-clear/semlink#2]
Hi,

This is a great resource, but looking through the code in the tools folder, they are full of absolute paths on someone's computer and refer to resources that do not seem to exist here. 

There seems to be no indication at all of how you can use the mappings files. Looking at them doesn't provide a lot of insight. It's pretty hard to use if there is no information.

I checked docs for older versions, but they refer to files that don't seem to exist anymore. 

Does anyone know of even just a simple example of using the mappings that exists anywhere and that matches with this version?

Thanks!
Alan

**Comment by kevincstowe** [comment]
Yes - apologies for the local versions, this was moved public maybe a little too soon! But I will be going through and updating things over the coming days. I'll have a more comprehensive readme and hopefully some example use-cases up today. If you've got other use cases you'd like to see, please post them here and we'll see what we can do.

One thing to note, we don't directly provide a mapping between FrameNet and PropBank - we map PB to VN, and then VN to FN. You can still retrieve PB - FN, but it requires moving through VN, which will introduce some noise. I'll include a sample to this effect.

**Comment by eahogue** [comment]
Thanks a lot! The new documentation is great!

Couple of points of clarification: 

1) So do I understand correctly that the instances files have standoff annotation for ontonotes, and that it includes the framenet frame and roleset IDs along with PB and VN? IOW, if you use the instances, does that obviate the need for mapping?

2) Do you have any idea how long the new VN to FN role mappings will take? I want to try using them all together.

Thanks again!
Alan
